extensions/podman/src/extension.ts

```diff
@@ -899,11 +911,18 @@ export async function deactivate(): Promise<void> {
   });
 }
 
+const PODMAN_MINIMUM_VERSION_FOR_USER_MODE_NETWORKING = '4.6.0';
+
+function isUserModeNetworkingSupported(podmanVersion: string) {
```

Comment:
Should add comments that this is windows only. maybe we should specific the function name that it's windows related since that's the only one that supports it.